code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3759,3 +3759,77 @@ datum
 			alch_strength = 0.4
 			description = "A warm, late night drink, usually enjoyed during long winter nights."
 			reagent_state = LIQUID
+
+		fooddrink/alcoholic/grenadine
```

Comment:
Possible future TODO: growable pomegranates -> pomegranate juice -> sugar + water + pomegranate juice recipe for grenadine.